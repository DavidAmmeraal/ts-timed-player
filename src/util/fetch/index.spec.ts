/**
 * index.spec.ts
 */

import * as fetchUtils from '.';

describe('util/fetch', () => {
  it('should match snapshot', () => {
    expect(fetchUtils).toMatchSnapshot();
  });
});