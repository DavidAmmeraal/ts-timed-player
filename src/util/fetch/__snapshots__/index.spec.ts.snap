// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`util/fetch should match snapshot 1`] = `
Object {
  "HttpError": [Function],
  "fetchWithHttpErrorHandler": [Function],
  "httpErrorHandler": [Function],
}
`;
